src/tests/ftest/util/mpio_test_base.py

```diff
@@ -65,9 +90,12 @@ def run_test(self, test_repo, test_name):
         # initialise test specific variables
         client_processes = self.params.get("np", '/run/client_processes/')
 
+        # create container
+        self._create_cont()
+
         try:
             # running tests
-            self.mpio.run_llnl_mpi4py_hdf5(
+            self.mpio.run_mpiio_tests(
```

Comment:
(pylint-no-value-for-parameter) No value for argument 'cont_uuid' in method call